src/root/layout.tt

```diff
@@ -81,7 +81,8 @@
 
       [% IF !hideHeader %]
         <div class="page-header">
-          [% IF c.user_exists && starUri; INCLUDE makeStar; " "; END; HTML.escape(title) %]
+          [% IF c.user_exists && starUri; INCLUDE makeStar; " "; END %]
+          [% IF titleHTML != ""; titleHTML; ELSE; HTML.escape(title); END %]
```

Comment:
I know it's "technically better" to do it that way, but in an entirely different context than perl-based template, I previously used an HTML sanitizer to "stringify" HTML strings into string titles. In case this can spark an idea for reducing exhausting repetitiveness.

Reply:
Unfortunately, I don't think this is possible with our current tools. I like the idea though :).